services/src/main/java/edu/unc/lib/dl/cdr/services/processing/XMLImportService.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.cdr.services.processing;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import edu.unc.lib.dl.acl.util.AgentPrincipals;
+import net.greghaines.jesque.Job;
+import net.greghaines.jesque.client.Client;
+
+/**
+ * A service for pushing a bulk-metadata-import job to a queue
```

Comment:
While this class has a `pushJobToQueue` method, I think it's primary purpose is "A service for performing import of bulk metadata updates" or something along those lines.

Reply:
it just seemed to me that the job is what is actually doing the work of importing the updates, but to appease the t-rex, i have changed it :-)

Reply:
yeah, i kind of waffled on whether to add this comment, it just seemed like the name of the class indicated that it was broader than putting something in a queue